sdk/provisioning/Azure.Provisioning.CloudMachine/src/CloudMachineInfrastructure.cs

```diff
@@ -32,54 +35,58 @@ public class CloudMachineInfrastructure : Infrastructure
     /// </summary>
     public BicepParameter PrincipalIdParameter => new BicepParameter("principalId", typeof(string));
 
-    /// <summary>
-    /// The common principalType parameter.
-    /// </summary>
-    public BicepParameter PrincipalTypeParameter => new BicepParameter("principalType", typeof(string));
+    ///// <summary>
+    ///// The common principalType parameter.
+    ///// </summary>
+    //public BicepParameter PrincipalTypeParameter => new BicepParameter("principalType", typeof(string));
```

Comment:
nit: delete commented code

Reply:
this is chris's code we agreed to comment out for now, but I think he will need it later.